### PR TITLE
Improve the API docs, module MD, and package MD

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
$title

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: xslt-ballerina/Module.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Module Overview
+
+This module provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
```
Comment (daneshk) [inline xslt-ballerina/Module.md:3]:
```suggestion
This module provides an API to transform the XML content to another XML/HTML/plain text using XSL transformations.
```

**Review by daneshk — APPROVED** [review]
LGTM

File: xslt-ballerina/Package.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
-For information on the operations, which you can perform with this module, see the below **Functions**. For examples on the usage of the operations, see the [XSLT Transformation Example](https://ballerina.io/learn/by-example/xslt-transformation.html).
\ No newline at end of file
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+### Useful Links
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
+- Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
+- Post all technical questions on Stack Overflow with the #ballerina tag.
```
Comment (MadhukaHarith92) [inline xslt-ballerina/Package.md:14]: Add link here.

File: xslt-ballerina/Package.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
 This package provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
 
-For information on the operations, which you can perform with this module, see the below **Functions**. For examples on the usage of the operations, see the [XSLT Transformation Example](https://ballerina.io/learn/by-example/xslt-transformation.html).
\ No newline at end of file
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+### Useful Links
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```
Comment (MadhukaHarith92) [inline xslt-ballerina/Package.md:12]: Add link here.

File: README.md [hunk 4]
```diff
@@ -87,6 +87,6 @@ All contributors are encouraged to read the [Ballerina Code of Conduct](https://
 
 ## Useful Links
 
-* Discuss about code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
+* Discuss the code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```
Comment (MadhukaHarith92) [inline README.md:90]:
```suggestion
* Discuss code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```

File: xslt-ballerina/Package.md (hunk 3 above)
Comment (MadhukaHarith92) [inline xslt-ballerina/Package.md:12]:
```suggestion
- Discuss code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```

Comment (MadhukaHarith92) [reply xslt-ballerina/Package.md:12]: Add link here.

Comment (BuddhiWathsala) [reply xslt-ballerina/Package.md]: Fixed in here https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/97a6cdaeece38447d139b67db168d614b8278682

File: README.md (hunk 4 above)
Comment (BuddhiWathsala) [reply README.md:90]: Fixed in here https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/97a6cdaeece38447d139b67db168d614b8278682

File: xslt-ballerina/Package.md (hunk 2 above)
Comment (BuddhiWathsala) [reply xslt-ballerina/Package.md]: Fixed in here https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/97a6cdaeece38447d139b67db168d614b8278682

File: README.md (hunk 4 above)
Comment (MadhukaHarith92) [inline README.md:90]:
```suggestion
* Discuss code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

File: README.md [hunk 5]
```diff
@@ -7,9 +7,9 @@ Ballerina XSLT Library
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-xslt/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-xslt)
 
-The XSLT library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+The XSLT library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
```
Comment (praneesha) [inline README.md:10]:
```suggestion
The `xslt` library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
```

File: README.md [hunk 6]
```diff
@@ -7,9 +7,9 @@ Ballerina XSLT Library
...
 
-This module provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
+This package provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
 
 For more information go to [The XSLT Module](https://ballerina.io/swan-lake/learn/api-docs/ballerina/xslt/).
```
Comment (praneesha) [inline README.md:14]:
```suggestion
For more information, go to the [`xslt` module](https://ballerina.io/swan-lake/learn/api-docs/ballerina/xslt/).
```

File: xslt-ballerina/Package.md [hunk 7]
```diff
@@ -1,5 +1,15 @@
...
+The XSLT library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
+</a> language.
 
 This package provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
 
-For information on the operations, which you can perform with this module, see the below **Functions**. For examples on the usage of the operations, see the [XSLT Transformation Example](https://ballerina.io/learn/by-example/xslt-transformation.html).
\ No newline at end of file
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```
Comment (praneesha) [inline xslt-ballerina/Package.md:9]:
```suggestion
To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Standard Library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

File: README.md (hunk 4 above)
Comment (BuddhiWathsala) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/18cfd06cc69a76b0cb2a6d6bff3e7c847b6c8304

File: README.md (hunk 5 above)
Comment (BuddhiWathsala) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/18cfd06cc69a76b0cb2a6d6bff3e7c847b6c8304

File: README.md (hunk 6 above)
Comment (BuddhiWathsala) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/18cfd06cc69a76b0cb2a6d6bff3e7c847b6c8304

File: xslt-ballerina/Package.md (hunk 7 above)
Comment (BuddhiWathsala) [reply xslt-ballerina/Package.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/18cfd06cc69a76b0cb2a6d6bff3e7c847b6c8304

**Review by praneesha — APPROVED** [review]

**Pull request merged**